client/a8c-for-agencies/components/form/field/index.tsx

```diff
@@ -6,7 +6,7 @@ import withErrorHandling from '../hoc/with-error-handling';
 import './style.scss';
 
 type Props = {
-	label: string;
+	label?: string;
```

Comment:
We have a new use case that do not require label so we need to make this optional.